miniml/compiler/compile.scm

```diff
@@ -151,9 +151,20 @@
 
    (type_name_with_args
     (LIDENT) : $1
-    (QUOTE LIDENT LIDENT) : $3
+    (variance QUOTE LIDENT LIDENT) : $3
     (LPAREN type_ignore RPAREN LIDENT) : $4)
 
+   (variance
+    (nonempty_variance) : '()
+    ( ) : '()
+   )
+
+   (nonempty_variance
+    (PLUS) : '()
+    (MINUS) : '()
+    (EQ) : '()
```

Comment:
Since when is `=` a variance? I don't see it in the manual.

Reply:
Apparently it exists in my brain only, as an explicit way to specify "invariant".